extensions-core/druid-basic-security/src/main/java/org/apache/druid/security/basic/CommonCacheNotifier.java

```diff
@@ -57,14 +58,7 @@
 {
   private static final EmittingLogger LOG = new EmittingLogger(CommonCacheNotifier.class);
 
-  private static final List<String> NODE_TYPES = Arrays.asList(
-      DruidNodeDiscoveryProvider.NODE_TYPE_BROKER,
-      DruidNodeDiscoveryProvider.NODE_TYPE_OVERLORD,
-      DruidNodeDiscoveryProvider.NODE_TYPE_HISTORICAL,
-      DruidNodeDiscoveryProvider.NODE_TYPE_PEON,
-      DruidNodeDiscoveryProvider.NODE_TYPE_ROUTER,
-      DruidNodeDiscoveryProvider.NODE_TYPE_MM
-  );
+  private static final List<NodeType> NODE_TYPES = Arrays.asList(NodeType.values());
```

Comment:
BTW not sure about this, if coordinator was skipped on purpose or by mistake. See https://github.com/apache/incubator-druid/pull/5099#discussion_r220149658. @jon-wei

Reply:
Reverted according to https://github.com/apache/incubator-druid/pull/5099#discussion_r220330380